highlights March 16th 2023 (#1725)
giantswarm/docs: src/content/changes/highlights/2023-03-16.md

---
date: 2023-03-16T11:00:00
title: Highlights for the week ending March 16, 2023
changes_categories:
  - Highlights
owner:
  - https://github.com/orgs/giantswarm/teams/sig-product
---

## General

To satisfy requirements that expand beyond Kubernetes, we now offer Crossplane as a managed solution. This is currently experimental and is known to cause potentially critical performance issues with the cluster it's running on. With this in mind, whilst we encourage you to experiment with it, we advise you to discuss this with your account engineer prior to installation.

> ## Apps
> 
> 
> ## User Interfaces

Reviewer: ```suggestion
## Apps

- [trivy-operator-app](https://github.com/giantswarm/trivy-operator-app) version [0.3.5](https://github.com/giantswarm/trivy-operator-app/blob/main/CHANGELOG.md#035---2023-03-14) updates to Trivy Operator v0.12.0.

## User Interfaces
```

---
date: 2023-03-16T11:00:00
title: Highlights for the week ending March 16, 2023
changes_categories:
  - Highlights
owner:
  - https://github.com/orgs/giantswarm/teams/sig-product
---

## General

To satisfy requirements that expand beyond Kubernetes, we now offer Crossplane as a managed solution. This is currently experimental and is known to cause potentially critical performance issues with the cluster it's running on. With this in mind, whilst we encourage you to experiment with it, we advise you to discuss this with your account engineer prior to installation.

## Apps

- [trivy-operator-app](https://github.com/giantswarm/trivy-operator-app) version [0.3.5](https://github.com/giantswarm/trivy-operator-app/blob/main/CHANGELOG.md#035---2023-03-14) updates to Trivy Operator v0.12.0.

> 

Reviewer 2: - [linkerd-bundle](https://github.com/giantswarm/linkerd-bundle) version [0.3.0](https://github.com/giantswarm/linkerd-bundle) With release 0.3.0 of our linkerd-bundle we released the app in the Giant Swarm Catalogue and made it public available. 
- [linkerd2-cni-app](https://github.com/giantswarm/linkerd2-cni-app) version [0.10.0](https://github.com/giantswarm/linkerd2-cni-app/blob/main/CHANGELOG.md#0100---2023-03-09) We upgraded  Linkerd to version 2.12.4 
- [linkerd-control-plane-app](https://github.com/giantswarm/linkerd-control-plane-app) version [0.11.0](https://github.com/giantswarm/linkerd-control-plane-app/blob/main/CHANGELOG.md#0110---2023-03-09) We upgraded  Linkerd to version 2.12.4 
- [linkerd-multicluster-app](https://github.com/giantswarm/linkerd-multicluster-app) version [0.10.0](https://github.com/giantswarm/linkerd-multicluster-app/blob/main/CHANGELOG.md#0100---2023-03-09) We upgraded  Linkerd to version 2.12.4
- [linkerd-multicluster-link-app](https://github.com/giantswarm/linkerd-multicluster-link-app) version [0.10.0](https://github.com/giantswarm/linkerd-multicluster-link-app/blob/main/CHANGELOG.md#0100---2023-03-09) We upgraded  Linkerd to version 2.12.4
- [linkerd-viz-app](https://github.com/giantswarm/linkerd-viz-app) version [0.09.0](https://github.com/giantswarm/linkerd-viz-app/blob/main/CHANGELOG.md#090---2023-03-09) We upgraded  Linkerd to version 2.12.4

> 

Reviewer 2: ```suggestion
- [linkerd-bundle](https://github.com/giantswarm/linkerd-bundle) version [0.3.0](https://github.com/giantswarm/linkerd-bundle) With release 0.3.0 of our linkerd-bundle we released the app in the Giant Swarm Catalogue and made it public available. 
- [linkerd2-cni-app](https://github.com/giantswarm/linkerd2-cni-app) version [0.10.0](https://github.com/giantswarm/linkerd2-cni-app/blob/main/CHANGELOG.md#0100---2023-03-09) We upgraded  Linkerd to version 2.12.4 
- [linkerd-control-plane-app](https://github.com/giantswarm/linkerd-control-plane-app) version [0.11.0](https://github.com/giantswarm/linkerd-control-plane-app/blob/main/CHANGELOG.md#0110---2023-03-09) We upgraded  Linkerd to version 2.12.4 
- [linkerd-multicluster-app](https://github.com/giantswarm/linkerd-multicluster-app) version [0.10.0](https://github.com/giantswarm/linkerd-multicluster-app/blob/main/CHANGELOG.md#0100---2023-03-09) We upgraded  Linkerd to version 2.12.4
- [linkerd-multicluster-link-app](https://github.com/giantswarm/linkerd-multicluster-link-app) version [0.10.0](https://github.com/giantswarm/linkerd-multicluster-link-app/blob/main/CHANGELOG.md#0100---2023-03-09) We upgraded  Linkerd to version 2.12.4
- [linkerd-viz-app](https://github.com/giantswarm/linkerd-viz-app) version [0.09.0](https://github.com/giantswarm/linkerd-viz-app/blob/main/CHANGELOG.md#090---2023-03-09) We upgraded  Linkerd to version 2.12.4

```

Author: @weatherhog if i commit this suggestion, the contribution from zach gets overwritted so I copy paste this into the file directly.

---
date: 2023-03-16T11:00:00
title: Highlights for the week ending March 16, 2023
changes_categories:
  - Highlights
owner:
  - https://github.com/orgs/giantswarm/teams/sig-product
---

## General

To satisfy requirements that expand beyond Kubernetes, we now offer Crossplane as a managed solution. This is currently experimental and is known to cause potentially critical performance issues with the cluster it's running on. With this in mind, whilst we encourage you to experiment with it, we advise you to discuss this with your account engineer prior to installation.

## Apps

- [trivy-operator-app](https://github.com/giantswarm/trivy-operator-app) version [0.3.5](https://github.com/giantswarm/trivy-operator-app/blob/main/CHANGELOG.md#035---2023-03-14) updates to Trivy Operator v0.12.0.

> - [linkerd-bundle](https://github.com/giantswarm/linkerd-bundle) version [0.3.0](https://github.com/giantswarm/linkerd-bundle) With release 0.3.0 of our linkerd-bundle we released the app in the Giant Swarm Catalogue and made it public available. 

Reviewer 3: ```suggestion
- [linkerd-bundle](https://github.com/giantswarm/linkerd-bundle) version [0.3.0](https://github.com/giantswarm/linkerd-bundle/releases/tag/v0.3.0) updates all linkerd components to the latest and publish the version to the public catalog. 
```

> - [linkerd2-cni-app](https://github.com/giantswarm/linkerd2-cni-app) version [0.10.0](https://github.com/giantswarm/linkerd2-cni-app/blob/main/CHANGELOG.md#0100---2023-03-09) We upgraded Linkerd to version 2.12.4.

Reviewer 3: ```suggestion
- [linkerd2-cni-app](https://github.com/giantswarm/linkerd2-cni-app) version [0.10.0](https://github.com/giantswarm/linkerd2-cni-app/blob/main/CHANGELOG.md#0100---2023-03-09) upgrades the component to latest Linkerd [`2.12.4`](https://github.com/linkerd/linkerd2/releases/tag/stable-2.12.4) version .
```

- [linkerd-control-plane-app](https://github.com/giantswarm/linkerd-control-plane-app) version [0.11.0](https://github.com/giantswarm/linkerd-control-plane-app/blob/main/CHANGELOG.md#0110---2023-03-09) We upgraded Linkerd to version 2.12.4. 
- [linkerd-multicluster-app](https://github.com/giantswarm/linkerd-multicluster-app) version [0.10.0](https://github.com/giantswarm/linkerd-multicluster-app/blob/main/CHANGELOG.md#0100---2023-03-09) We upgraded Linkerd to version 2.12.4.
- [linkerd-multicluster-link-app](https://github.com/giantswarm/linkerd-multicluster-link-app) version [0.10.0](https://github.com/giantswarm/linkerd-multicluster-link-app/blob/main/CHANGELOG.md#0100---2023-03-09) We upgraded Linkerd to version 2.12.4.
- [linkerd-viz-app](https://github.com/giantswarm/linkerd-viz-app) version [0.09.0](https://github.com/giantswarm/linkerd-viz-app/blob/main/CHANGELOG.md#090---2023-03-09) We upgraded Linkerd to version 2.12.4.

> 

Reviewer 3: ```suggestion

- [Linkerd](https://github.com/linkerd/linkerd2) version [`2.12.4`](https://github.com/linkerd/linkerd2/releases/tag/stable-2.12.4) fixes a memory leak in the destination controller, and also includes other bug fixes for the Linkerd control plane, CLI, and extensions. [More information here](https://github.com/linkerd/linkerd2/releases/tag/stable-2.12.4). All dependent components have been upgraded too: 
  - [linkerd2-cni-app](https://github.com/giantswarm/linkerd2-cni-app) version [0.10.0](https://github.com/giantswarm/linkerd2-cni-app/blob/main/CHANGELOG.md#0100---2023-03-09).
  - [linkerd-control-plane-app](https://github.com/giantswarm/linkerd-control-plane-app) version [0.11.0](https://github.com/giantswarm/linkerd-control-plane-app/blob/main/CHANGELOG.md#0110---2023-03-09).
  - [linkerd-multicluster-app](https://github.com/giantswarm/linkerd-multicluster-app) version [0.10.0](https://github.com/giantswarm/linkerd-multicluster-app/blob/main/CHANGELOG.md#0100---2023-03-09) We upgraded Linkerd to version 2.12.4.
  - [linkerd-multicluster-link-app](https://github.com/giantswarm/linkerd-multicluster-link-app) version [0.10.0](https://github.com/giantswarm/linkerd-multicluster-link-app/blob/main/CHANGELOG.md#0100---2023-03-09).
  - [linkerd-viz-app](https://github.com/giantswarm/linkerd-viz-app) version [0.09.0](https://github.com/giantswarm/linkerd-viz-app/blob/main/CHANGELOG.md#090---2023-03-09).

- The [linkerd-bundle](https://github.com/giantswarm/linkerd-bundle) version [0.3.0](https://github.com/giantswarm/linkerd-bundle/releases/tag/v0.3.0) updates all Linkerd components to the latest and publish the version to the public catalog. 

```

Author: @pipo02mix can we resolve this, since this suggestion is overriden by the new one you proposed below (and it's already committed?)
